app/src/main/java/com/yacgroup/yacguide/CountryActivity.kt

```diff
@@ -67,4 +78,26 @@ class CountryActivity : UpdatableTableActivity() {
     override fun getLayoutId(): Int {
         return R.layout.activity_country
     }
+
+    /*
+     * Check whether the "What's New" dialog is shown or not. This is decided by comparing
+     * the version stored in the settings with the current app version.
+     */
+    private fun _init() {
+        val settingsKeyVerCode = "version_code"
```

Comment:
Please declare this setting in `strings.xml`. All other settings are declared there, too.

Reply:
The `strings.xml` is not a good place because it will be translated, which is not intended. I recommend to created a new resource file which only handles the preference keys.